drivers/hmis_external_apis/app/models/hmis_external_apis/external_id.rb

```diff
@@ -0,0 +1,16 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# FIXME: Any chance this would be used by processes outside of this driver?
```

Comment:
It will be references outside the driver, but that's okay. The hmis driver has an implicit "dependency" on this driver.